src/js/modules/infragistics.ui.editors.js

```diff
@@ -4678,6 +4673,18 @@
 				this.options.maxDecimals = this._getOptionOrRegionalValue("minDecimals");
 			}
 		},
+		_getValueBetweenMinMax: function(value) {
+			// N.A. 7 November 2018, Bug #1834, Initial value that is null, should not be overwritten by the min/max values.
+			if (value !== this.options.nullValue) {

```

Comment:
And if `allowNullValue` is not enabled?

Reply:
Thanks, I've fixed that and I've added a test.